migrations/20240618124843-change-review-task-history-target-model-field.ts

```diff
@@ -0,0 +1,19 @@
+import { Db } from "mongodb";
+
```

Comment:
question: will we need to change the table name as well?

Reply:
Yes, we do need. To do that in my local environment I exported the `claimreviewtasks` document, created a new one named `reviewtasks` and imported the data

Reply:
Can't we have a migration to rename the collection?